pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -453,21 +351,21 @@ func envIsMatchedByRule(envVariable string, rules []EnvRuleConfig) bool {
 	return false
 }
 
-func (pe *StandardSecurityPolicyEnforcer) expandMatchesForContainerIndex(index int, idToAdd string) {
```

Comment:
A lot of this refactoring/renaming stuff would have also been good to split out. Makes reviewing the actual meat of these changes a lot easier if the PR is smaller.